cmd/experimental/kjobctl/hack/tools/go.mod

```diff
@@ -0,0 +1,67 @@
+module sigs.k8s.io/kueue/cmd/experimental/kjobctl/hack/tools
```

Comment:
I think we need to add a rule for this module in `.github/dependabot.yml`

Reply:
#2450